entries/jQuery.when.xml

```diff
@@ -3,60 +3,69 @@
   <title>jQuery.when()</title>
   <signature>
     <added>1.5</added>
-    <argument name="deferreds" type="Deferred">
-      <desc>One or more Deferred objects, or plain JavaScript objects.</desc>
+    <argument name="value" type="Anything" optional="true">
+      <desc>A Promise-like object to resolve a Deferred object, or any value to encapsulate within a Deferred object.</desc>
+    </argument>
+  </signature>
+  <signature>
+    <added>1.5</added>
+    <argument name="values" type="Anything">
+    <desc>Two or values to resolve or encapsulate with Deferred objects for <code>Promise.all</code>-like fulfillment aggregation.</desc>
     </argument>
   </signature>
   <desc>Provides a way to execute callback functions based on one or more objects, usually <a href="/category/deferred-object/">Deferred</a> objects that represent asynchronous events.</desc>
   <longdesc>
-    <p>If a single Deferred is passed to <code>jQuery.when()</code>, its Promise object (a subset of the Deferred methods) is returned by the method. Additional methods of the Promise object can be called to attach callbacks, such as <a href="/deferred.then/"><code>deferred.then</code></a>. When the Deferred is resolved or rejected, usually by the code that created the Deferred originally, the appropriate callbacks will be called. For example, the jqXHR object returned by <code>jQuery.ajax()</code> is a Promise-compatible object and can be used this way:</p>
+    <p>If no arguments are provided, <code>jQuery.when()</code> will return a resolved Promise.</p>
```

Comment:
I wonder if we need to create some other term than `Promise` since what's returned here is a `Deferred.promise` which is consumed a bit differently and has methods like `.done`, etc. The `types.xml` file does use `Promise` for this but now that we consume ES6 Promise it's ambiguous.